.github/workflows/build.yaml

```diff
@@ -27,3 +27,27 @@ jobs:
           echo "/usr/local/kubebuilder/bin" >> $GITHUB_PATH
           make toolchain build test
         working-directory: operator
+  build-cli:
+    strategy:
+      matrix:
+        os: ["darwin"]
```

Comment:
Can we standardize on linux?

Reply:
The release process builds for macOS, if we validate here Linux we might run into issues when releasing. Are you suggesting build for both here?